plugins/modules/notification/cisco_webex.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: cisco_webex
+short_description: Send a webexmsg to a Cisco Webex Teams Room or Individual.
+description:
+    - Send a webexmsg to a Cisco Webex Teams Room or Individual with options to control the formatting.
+version_added: "2.3"
+author: Drew Rusell (@drew-russell)
+notes:
+  - The C(recipient_id) type must be valid for the supplied C(recipient_id).
+  - Full API documentation can be found at U(https://developer.ciscospark.com/endpoint-webexmsgs-post.html).
+
+options:
+
+  recipient_type:
+    description:
+       - The request parameter you would like to send the webexmsg to.
+       - Messages can be sent to either a room or individual (by ID or E-Mail).
+    required: True
```

Comment:
```suggestion
    required: yes
```
etc. for the other parameters
it's from the doc